work-process.md

```diff
@@ -63,6 +63,77 @@ Activities that need to be performed so this user story can be moved to `Done`.
 In the "Vote" step (the User Stories that have ``✋ [vote]`` prefix), Scrum Master assigns the User Story to people from whom s/he wants to receive Story Point estimations. Once these people post their story point estimation, they unassign themselves from the User Story. The Scrum Master then adds the agreed numbered Story Point label to the title, removes the ``✋ [vote]`` prefix and moves the User Story to the top of the pipeline. This User Story can now be added to the next sprint.
 
 
+## Pull Requests and Commits
+
+While working on User Stories we often need to change code or text that we keep in GitHub. These changes are done with [git commits](https://help.github.com/articles/github-glossary/#commit) which are presented to others for reviewal with [Pull Requests](https://help.github.com/articles/about-pull-requests/).
+
+To make sure that your commits and your Pull Requests contain everything for a successful review, follow the conventions below:
+
+### Git commit
+
+A single commit contains only a single "unit of work". For example, if you are adding a new section to this Handbook, but then notice a typo in some other section, the typo should be in a separate commit.
+
+The commit message should look like this:
+
+> Short, up to 50 characters summary of changes
+>
+> More detailed explanatory text. Wrap it to about 72 characters or so.
+> The blank line separating the summary from the body is critical;
+> tools like GitHub can get confused if you omit it.
+>
+> Further paragraphs come after blank lines.
+>
+> It is critical that you explain *why* you did the change. Don't state
+> the obvious "File updated", "Text removed". Imagine you are reading
+> this message after two years; how would you explain your future self
+> the reason for this change?
+>
+>   - Bullet points are okay, too
+>
+>   * Typically a hyphen or asterisk is used for the bullet,
+>     preceded by a single space, with blank lines in
+>     between.
+>
+> At the bottom of the commit message, reference the Issue(s) to which
+> this commit belongs, in the following format:
+>
+> Refs #123.
+> Refs niteoweb/operations#123.
+
+
+A few examples of excellent commits:
+ * TODO
+ * TODO
+ * TODO
+
+
+### Pull Request
+
+When your commit is ready to be applied to code/text inside GitHub, you need to present it with a Pull Request.
+Similarly to commits, a Pull Request should only include commits on a single topic. For example, if you are adding a new section to this Handbook, but then notice that some other section is also missing, that second section should be in a separate commit.
```

Comment:
Is it meant to end with 'separate commit' or did you mean 'separate PR'? The surrounding text and paragraph below suggest the latter.

Reply:
PR